lib/osf-components/addon/components/editable-field/institutions-manager/template.hbs

```diff
@@ -14,5 +12,5 @@
     bindReload=(action (mut this.reloadList))
     node=this.node
     user=this.currentUser.user
-    isSaving=this.save.isRunning
```

Comment:
Any reason you renamed `isSaving`? `isSaving` is expected on the editable managers interface https://github.com/CenterForOpenScience/ember-osf-web/blob/079d2eb3fb4e6f55b4694f4cd30401c7db909230/lib/osf-components/addon/components/editable-field/template.hbs#L70
https://github.com/CenterForOpenScience/ember-osf-web/blob/079d2eb3fb4e6f55b4694f4cd30401c7db909230/lib/osf-components/addon/components/editable-field/template.hbs#L59

Reply:
The intention was to decouple the manager from the consuming services a bit more. Instead of using manager's `isSaving` property to determine whether the component should be disabled, the manager should yield a `shouldDisable` property for this purpose, because it isn't always true that the widget should be disabled when saving.
But it seems it's more tangled than I thought. I can change it back.

Reply:
yes, it's a bit messy because the property is used in multiple places. So, the way I understand this, the manager's concern should be exposing data and ways to alter it. The consumer (display component) decides what to do with the data for UI purposes. So naming `shouldDisable` would instead mean coupling instead of decoupling manager from the display component?

Reply:
It definitely can go both ways. For example, if manager is something constant (meaning the manager with the same shape will be used in different places), then it would be coupling instead of decoupling. But the problem now is the consuming component (`institutions-select-list`) is constant and we need new managers to handle new data-loading and data-saving behaviors.

Reply:
this is a bit beyond the point I was trying to make with the first comment. Making this change on the `editable-field/institutions-manager` also means you'll need to change both references (mentioned above) and that would break other editable field managers which expose `isSaving` property.